lwcomponent/component_test.go

```diff
@@ -0,0 +1,152 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package lwcomponent_test
+
+import (
+	"io"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/lacework/go-sdk/lwcomponent"
+	"github.com/stretchr/testify/assert"
+	capturer "github.com/zenizh/go-capturer"
```

Comment:
Instead of adding a dependency, could we create an internal go package out of:
* https://github.com/lacework/go-sdk/blob/main/lwlogger/logger_test.go#L232
* https://github.com/lacework/go-sdk/blob/main/cli/cmd/capture_output_from_test.go#L29-L31

Reply:
Will do!

Reply:
Done!